pyproject.toml

```diff
@@ -11,16 +11,16 @@ documentation = "https://electionguard-web-api.readthedocs.io/"
 
 [tool.poetry.dependencies]
 python = "~=3.8"
-electionguard = "*"
-fastapi = "*"
-uvicorn = "*"
+electionguard = "~1.1"
+fastapi = "~0.61"
+uvicorn = "~0.11"
 
 [tool.poetry.dev-dependencies]
-black = { version = "*", allow-prereleases = true }
-mkdocs = "*"
-mypy = "*"
-pylint = "*"
-pytest = "*"
+black = { version = "^20", allow-prereleases = true }
```

Comment:
We actually might want to be more specific with this one in particular because the changes in 20 I already had issues with.

Reply:
The caret notation should keep it locked to v20 unless we explicitly upgrade, but will allow minor upgrades.  Does that work for you in this case?

Reply:
It's minor versions. The change on the last a->b had the files flipping back and forth.